source/system/math.ooc

```diff
@@ -184,8 +184,8 @@ extend Float {
 	greaterThan: func (other: This, tolerance := This epsilon) -> Bool { this > other && !this equals(other, tolerance) }
 	lessOrEqual: func (other: This, tolerance := This epsilon) -> Bool { !this greaterThan(other, tolerance) }
 	greaterOrEqual: func (other: This, tolerance := This epsilon) -> Bool { !this lessThan(other, tolerance) }
-	linearInterpolation: func (a: This, b: This) -> This { (this * (b - a)) + a }
-	inverseLinearInterpolation: func (a: This, b: This) -> This { (this - a) / (b - a) }
+	mix: static func (a, b, ratio: This) -> This { (ratio * (b - a)) + a }
+	inverseMix: static func (a, b, ratio: This) -> This { (ratio - a) / (b - a) }
```

Comment:
Renamed the functions and made them static with ratio at the end. The old pattern cannot be used for anything else than Float because the owner was used as the ratio that is always a Float.